third_party/move/move-model/src/builder/mod.rs

```diff
@@ -11,8 +11,22 @@ pub(crate) mod module_builder;
 pub(crate) fn pluralize(s: &str, n: usize) -> String {
     // Should add special cases here as we come along them
     if n != 1 {
-        s.to_owned() + "s"
+        if s.ends_with('y') {
+            s[0..s.len() - 1].to_string() + "ies"
+        } else {
+            s.to_owned() + "s"
+        }
     } else {
         s.to_owned()
     }
 }
+
+pub(crate) fn ith_str(n: usize) -> String {
+    match n {
+        0 => panic!("cannot be 0"),
+        1 => "1st".to_string(),
+        2 => "2nd".to_string(),
+        3 => "3rd".to_string(),
```

Comment:
Note that "23rd", "33rd", etc., would also be special cases.  Perhaps you should just use crate "num_ordinal":
```
use num_ordinal::{Osize};
fn ith_str(n: usize) -> String {
   Osize::from1(n).to_string()
}
```

Reply:
Probably not worth it, though, since that's a new crate.

Reply:
Agree, lets get to there when we hit those large numbers.